doc/nrf/libraries/nrf_security/doc/driver_config.rst

```diff
@@ -60,27 +64,33 @@ To enable cipher modes, set one or more of the following Kconfig options:
-+-----------------------+---------------------------------------------------------------------+----------------------------------------------------------------------+
-| OFB                   | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_OFB_CC3XX`            | Not supported                                                        |
-+-----------------------+---------------------------------------------------------------------+----------------------------------------------------------------------+
-| XTS                   | Not supported                                                       | Not supported                                                        |
-+-----------------------+---------------------------------------------------------------------+----------------------------------------------------------------------+
-| Stream cipher         | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_STREAM_CIPHER_CC3XX`  | :kconfig:option:`CONFIG_PSA_CRYPTO_DRIVER_ALG_STREAM_CIPHER_OBERON`  |
-+-----------------------+---------------------------------------------------------------------+----------------------------------------------------------------------+
+You can use the following table to check the cipher algorithm support of each driver:
+
++-----------------------+---------------------------+----------------------------+
+| Cipher mode           | nrf_cc3xx driver support  | nrf_oberon driver support  |
++=======================+===========================+============================+
+| ECB no padding        | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
+| CBC no padding        | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
+| CBC PKCS#7 padding    | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
+| CFB                   | Not supported             | Not supported              |
++-----------------------+---------------------------+----------------------------+
+| CTR                   | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
+| OFB                   | Supported                 | Not supported              |
++-----------------------+---------------------------+----------------------------+
+| XTS                   | Not supported             | Not supported              |
++-----------------------+---------------------------+----------------------------+
+| Stream cipher         | Supported                 | Supported                  |
++-----------------------+---------------------------+----------------------------+
```

Comment:
If you have a tool for editing these tables please put a lot of room for new Cipher modes with long names so I don't have to manually extend the rows later.

If you are doing this manually nevermind.